src/libLookup/Lookup.cpp

```diff
@@ -1218,6 +1218,237 @@ bool Lookup::ProcessSetSeedPeersFromLookup(const vector<unsigned char>& message,
     return true;
 }
 
+bool Lookup::AddMicroBlockToStorage(const uint64_t& blocknum,
+                                    const MicroBlock& microblock)
+{
+    if (!LOOKUP_NODE_MODE)
```

Comment:
We can remove this check if we already know where this function would be called.